CHANGES.rst

```diff
@@ -81,6 +79,14 @@ scripts
 - Removed many non-working and out-dated scripts. Including
   many scripts that were replaced by ``strun``. [#8619]
 
+set_telescope_pointing
+----------------------
+
+- replace usage of ``copy_arrays=True`` with ``memmap=False`` [#8660]
+
+- Refactored separate modes into submodules instead of inheriting from a base class.
+  Moved non-JWST-specific code to stcal. [#8613]
+
```

Comment:
Looking at #8613, I think this comment was supposed to go under the outlier detection heading.

Reply:
This isn't my change.  That bullet is under `set_telescope_pointing` in the master repo.  It only appears as my edit because of git's problem with CHANGES.rst always claiming merge conflicts:

https://github.com/spacetelescope/jwst/blob/9667e6c8fcdf316aa558544afa54c7b8528a5540/CHANGES.rst?plain=1#L70

Reply:
Thanks, it should be fixed on main now.